samples/apps/auth-api-webapp-react/README.md

```diff
@@ -9,22 +9,24 @@
 
 ## Running the sample
 
-1. You will need an [Open AI Key](https://openai.com/api/) or
+1. Create a new copy of `.env.development` and save it as `.env` next to the original. 
+   - The `.env` file will be ignored by git and will not be checked in by default.
+1. You will need an [Open AI Key](https://platform.openai.com/account/api-keys) or
```

Comment:
pls undo the "1." "1." "1." change. I know that markdown renders numbers automatically, but we should keep markdown docs readable also without the need to render them (which I believe is one of the goals of MD :-))

Reply:
@dluc good point! Changed!